scripts/deploy_mock_governance.ts

```diff
@@ -4,29 +4,29 @@
 import { ethers, providers } from 'ethers'
 
 import {
-  TestTrustTokenFactory,
-  MockTimeLockFactory,
-  GovernorAlphaFactory,
-  OwnedUpgradeabilityProxyFactory,
-} from '../build'
+  TestTrustToken__factory,
```

Comment:
is there a reason we need to use this new double underscore? Not really an issue just curious

Reply:
We updated typechain and new version has namings like that to avoid conflicts like wi had with LoanToken/LoanTokenFactory